apps/models.go

```diff
@@ -281,10 +281,11 @@ func NewApp(p ManifestInputs) (*App, error) {
 		}
 	}
 
-	refreshInterval, err := time.ParseDuration(p.RefreshInterval)
+	refreshIntervalNanoseconds, err := time.ParseDuration(p.RefreshInterval)
 	if err != nil {
 		return nil, err
 	}
+	refreshIntervalHours := int64(refreshIntervalNanoseconds / time.Hour)
```

Comment:
Why would you do all this if it's already specified in hours?

Reply:
Per our discussion, this is here to maintain manifest schema syntax of `refreshInterval: 5h`. In next sweep of updating all app manifests, will update to `refreshIntervalHours: 5` and will be able to remove this then.